src/o365/outlook/commands/report/outlook-report-getmailboxusagestorage.ts

```diff
@@ -0,0 +1,38 @@
+import commands from '../../commands';
+import PeriodBasedReport from '../../../base/PeriodBasedReport';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+class OutlookGetMailboxUsageStroageCommand extends PeriodBasedReport {
+  public get name(): string {
+    return `${commands.OUTLOOK_USAGE_GETMAILBOXUSAGESTORAGE}`;
+  }
+
+  public get usageEndpoint(): string {
+    return 'getMailboxUsageStorage';
+  }
+
+  public get description(): string {
+    return 'Get the amount of storage used in your organization.';
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    log(vorpal.find(this.name).helpInformation());
+    log(
+      `  Examples:
+      
+    Gets the amount of storage used in your organization for the last week
```

Comment:
Could you add `mailbox` in front of storage in the examples? It might be a bit more clear for noobs like me. Thanks!